Upstream bug causes linux compile errors

There's a bug in the included Clipper version which was causing the following compile error for me:

```
../../../addons/obj/linux64/Release/ofxClipper/src/ofxClipper.o: In function `ofxClipper::ReversePolyline(ofPolyline&)':
ofxClipper.cpp:(.text+0xeb5): undefined reference to `ClipperLib::ReversePolygon(std::vector<ClipperLib::IntPoint, std::allocator<ClipperLib::IntPoint> >&)'
collect2: error: ld returned 1 exit status
```

I discovered a fix before researching and finding out that it's been fixed awhile upstream.  I can open a pull request with my fix (a couple of characters in clipper.cpp) or alternatively I can try an upgrade to 4.10 (the last 4.n update) and see if that works.  Do you have preferences?

--- a/libs/clipper/src/cpp/clipper.cpp
+++ b/libs/clipper/src/cpp/clipper.cpp
@@ -2971,7 +2971,7 @@ void Clipper::JoinCommonEdges()
 }
 //------------------------------------------------------------------------------
 
-void ReversePolygons(Polygon& p)
+void ReversePolygon(Polygon& p)
 {
   std::reverse(p.begin(), p.end());
 }
@@ -2980,7 +2980,7 @@ void ReversePolygons(Polygon& p)
 void ReversePolygons(Polygons& p)
 {
   for (Polygons::size_type i = 0; i < p.size(); ++i)
-    ReversePolygons(p[i]);
+    ReversePolygon(p[i]);
 }
 
 //------------------------------------------------------------------------------

